src/main/java/me/moros/bending/util/methods/EntityMethods.java

```diff
@@ -110,16 +110,17 @@ public static boolean isOnGround(@NonNull Entity entity) {
    */
   public static double distanceAboveGround(@NonNull Entity entity) {
     int maxHeight = entity.getWorld().getMaxHeight();
+    int minHeight = entity.getWorld().getMinHeight();
     AABB entityBounds = AABBUtils.entityBounds(entity).grow(new Vector3d(0, maxHeight, 0));
```

Comment:
You want to calculate delta height (maxHeight - minHeight) and use that for the bounding box expansion.